### PR TITLE
[KYUUBI #3973] Fix a conf fallback for `kyuubi-ctl list server` when kyuubi-defaults.conf use deprecated zk key

### PR DESCRIPTION
**Pull request by CavemanIV**
### _Why are the changes needed?_

To fix https://github.com/apache/incubator-kyuubi/issues/3973, we found similar problem during upgrade from 1.5 to 1.6

### _How was this patch tested?_
Manual test on internal cluster, UT will be added later on

- [ ] Add some test cases that check the changes thoroughly including negative and positive cases if possible

- [ ] Add screenshots for manual tests if appropriate

- [ ] [Run test](https://kyuubi.apache.org/docs/latest/develop_tools/testing.html#running-tests) locally before make a pull request

### CODE REVIEW

File: kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/Command.scala [hunk 1]
```diff
@@ -57,11 +57,13 @@ abstract class Command[T](cliConfig: CliConfig) extends Logging {
   private def useDefaultPropertyValueIfMissing(): CliConfig = {
     var arguments: CliConfig = cliConfig.copy()
     if (cliConfig.zkOpts.zkQuorum == null) {
-      conf.getOption(HA_ADDRESSES.key).foreach { v =>
```
Comment (pan3793) [inline kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/Command.scala:60]: maybe it can be simply fixed by `conf.get(HA_ADDRESSES)`

Comment (CavemanIV) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/Command.scala:60]: coding style consistent, done

Comment (pan3793) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/Command.scala:60]: Have you try my suggestion? The `KyuubiConf` system should support fallback mechanism

Comment (pan3793) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/Command.scala:60]: for code style, it can be quickly fixed by `dev/reformat`

Comment (CavemanIV) [reply kyuubi-ctl/src/main/scala/org/apache/kyuubi/ctl/cmd/Command.scala:60]: right `conf.get` is simpler and easy. I didn't notice `.get` and `getOption` have different implementation, thought fallback not working

**Review by cxzl25 — APPROVED** [review]

**Review by pan3793 — APPROVED** [review]

**Comment by pan3793** [comment]
Thanks, merged to master/1.6

**Pull request closed without merging**